contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -1019,114 +1139,144 @@ namespace eosiosystem {
                             const string& img_url, const string& bio, const string& country, const string& telegram,
                             const string& website, const string& linkedin);
 
-       [[eosio::action]]
-       void editproposer(name account, const string& first_name, const string& last_name,
-                             const string& img_url, const string& bio, const string& country, const string& telegram,
-                             const string& website, const string& linkedin);
+         [[eosio::action]]
```

Comment:
@quocle108 can we undo these cosmetic changes made to the WPS code please? They aren't related to the PR and just add confusion